client/src/components/AddCommentModal.js

```diff
@@ -0,0 +1,212 @@
+      newLabels.push(addLabel);
+      setLabels(newLabels);
+      setAddLabel('');
+    }
+  }
+
+  return (
+    <div>
+      <IconButton
+        id='addTicket'
+        edge='start'
+        className={classes.menuButton}
+        color='inherit'
+        aria-label='add ticket'
+        onClick={handleOpen}
+      >
+        <AddCommentIcon />
+      </IconButton>
+
+      <Modal
+        aria-labelledby='add-comment-title'
+        aria-describedby='transition-modal-description'
+        className={classes.modal}
+        open={open}
+        onClose={handleClose}
+        closeAfterTransition
+        BackdropComponent={Backdrop}
+        BackdropProps={{
+          timeout: 500,
+        }}
+      >
+        <Fade in={open}>
+          <div className={classes.paper}>
+            {error && (
+              <span id='formError' style={{ color: 'red' }}>
+                Make sure that everything is correct!
+              </span>
+            )}
+            <form id='addTicketForm'>
+              <fieldset id='ticketFieldSet' className={classes.form}>
+                <h3 id='add-ticket-title'>Add ticket</h3>
+                <label htmlFor='userEmail' className='formItem'>
+                  Email:
+                </label>
+                <input
+                  className='formItem'
+                  type='email'
+                  id='userEmail'
+                  name='userEmail'
+                  value={userEmail}
+                  onChange={(event) =>
+                    handleChange(setUserEmail, event.target.value)
+                  }
+                />
+
+                <label htmlFor='title' className='formItem'>
+                  Title:
+                </label>
+                <input
+                  className='formItem'
+                  type='text'
+                  id='title'
+                  name='title'
+                  value={title}
+                  onChange={(event) =>
+                    handleChange(setTitle, event.target.value)
+                  }
+                />
+
+                <label htmlFor='content' className='formItem'>
+                  Cotent:
+                </label>
+                <textarea
+                  className='formItem'
+                  id='content'
+                  name='content'
+                  rows='10'
+                  cols='50'
+                  value={content}
+                  onChange={(event) =>
+                    handleChange(setContent, event.target.value)
+                  }
+                />
+
+                <span>
+                  <label htmlFor='labels' className='formItem'>
+                    Labels:
+                  </label>
+                  {labels.map((label) => (
+                    <span key={label} className='labelsForm'>
+                      {label}
+                    </span>
+                  ))}
+                </span>
+                <span id='spanLabels'>
```

Comment:
Using `id` here indicates that this component could not be used more then once.

Reply:
The naming was bad, this span holds the input and button to add labels, so its used only once